src/EditorFeatures/Core/SplitComment/SplitCommentCommandHandler.cs

```diff
@@ -151,12 +151,12 @@ private static bool MatchesCommentStart(string commentStart, ITextSnapshotLine l
             ITextBuffer textBuffer,
             SnapshotSpan selectionSpan)
         {
-            var syntaxKinds = document.LanguageServices.GetRequiredService<ISyntaxKindsService>();
+            var syntaxKinds = document.ProjectServices.GetRequiredService<ISyntaxKindsService>();
```

Comment:
We have both document.ProjectServices and document.Project.Services? Do we really need both? Is there a shortage of `.`s these days?

Reply:
so this is not a normal document.  this is the internal ParsedDocument helper guy.  So this is how that guy bridges to this new api.